runtime/opt/taupage/runtime/Docker.py

```diff
@@ -95,6 +96,15 @@ def wait_for_local_planb_tokeninfo():
     return None
 
 
+def get_docker_command(config: dict):
+    '''get the docker command to use'''
+    cuda_device_files = glob.glob('/dev/nvidia*')
```

Comment:
Would it make sense to list the exact requirements here?
```
/dev/nvidia[0-9]+  
/dev/nvidiactl
/dev/nvidia-uvm
```
Although, I'm happy with the current implementation as a first step.

Does adding some logging for debugging startup make sense?